.github/workflows/pull-request.yml

```
@@ -9,55 +9,64 @@ jobs:
   CodeRabbit-Approval:
     name: Validate CodeRabbit.ai Approval
     runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
     steps:
-      - name: Wait for CodeRabbit.ai Approval
+      - name: Validate CodeRabbit.ai Approval
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:10: Double quote to prevent globbing and word splitting

(shellcheck)